src/edfi_amt_data_lake/parquet/amt/rls/rls_staff_classification_descriptor_scope_list/main.py

```diff
@@ -4,17 +4,28 @@
 # See the LICENSE and NOTICES files in the project root for more information.
 
 import pandas as pd
-from decouple import config
 
+from edfi_amt_data_lake.helper.data_frame_generation_result import (
+    data_frame_generation_result,
+)
 from edfi_amt_data_lake.helper.helper import get_descriptor_mapping_config
 from edfi_amt_data_lake.parquet.Common.pandasWrapper import (
+    create_empty_data_frame,
+    create_parquet_file,
     jsonNormalize,
     renameColumns,
-    saveParquetFile,
 )
 
+RESULT_COLUMNS = [
+    'AuthorizationScopeName',
+    'CodeValue'
+]
+
 
-def rls_staff_classification_descriptor_scope_list_dataframe(school_year) -> pd.DataFrame:
+@create_parquet_file
+def rls_staff_classification_descriptor_scope_list_dataframe(
```

Comment:
It's not working, it requires three parameters and now it has just one.
`rls_staff_classification_descriptor_scope_list_dataframe() takes 1 positional argument but 3 were given`